### PR TITLE
Add companion watchOS test app targets in GDTCCTWatchOSTestApp

### PR DESCRIPTION
**Pull request by doudounan**
- Add an iOS test app target for companion watchOS test app

- Add a companion watchOS test app watchkit target

- Add a companion watchOS test app watchkit extension target with interface controller to test upload gdtcct events.

### CODE REVIEW

File: GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSCompanionTestAppExtension/ExtensionDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  func applicationDidBecomeActive() {
+    // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+  }
+
+  func applicationWillResignActive() {
+    // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
+    // Use this method to pause ongoing tasks, disable timers, etc.
+  }
```
Comment (chliangGoogle) [inline GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSCompanionTestAppExtension/ExtensionDelegate.swift:31]: can remove all these code if not used.

File: GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTiOSTestAppForCompanionWatchApp/SceneDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  func sceneWillEnterForeground(_ scene: UIScene) {
+    // Called as the scene transitions from the background to the foreground.
+    // Use this method to undo the changes made on entering the background.
+  }
+
+  func sceneDidEnterBackground(_ scene: UIScene) {
+    // Called as the scene transitions from the foreground to the background.
+    // Use this method to save data, release shared resources, and store enough scene-specific state information
+    // to restore the scene back to its current state.
+  }
```
Comment (chliangGoogle) [inline GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTiOSTestAppForCompanionWatchApp/SceneDelegate.swift:55]: Remove these as well.

File: GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTiOSTestAppForCompanionWatchApp/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Google
```
Comment (chliangGoogle) [inline GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTiOSTestAppForCompanionWatchApp/ViewController.swift:2]: * Copyright 2020 Google LLC 

The same as all above files.

File: GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/Podfile [hunk 4]
```diff
@@ -14,3 +14,13 @@ target 'GDTCCTWatchOSIndependentTestAppWatchKitExtension' do
   pod 'GoogleDataTransportCCTSupport', :path => '../../'
 
 end
+
+target 'GDTCCTWatchOSCompanionTestAppExtension' do
+  platform :watchos, '6.0'
+
+  # Pods for GDTCCTWatchOSCompanionTestAppExtension
+  pod 'SwiftProtobuf', '~> 1.0'
```
Comment (chliangGoogle) [inline GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/Podfile:22]: Just curious, you guys need this one?

Comment (doudounan) [reply GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/Podfile:22]: This one is only used in test app to generate swift code from proto.

**Pull request merged**